save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestExecutionController.kt

```diff
@@ -131,6 +131,14 @@ class TestExecutionController(private val testExecutionService: TestExecutionSer
         testExecutionService.assignAgentByTest(agentContainerId, testDtos)
     }
 
+    /**
+     * @param crashedAgents the list of agents, which weren't sent heartbeats for a some time and are considered as crashed
+     */
+    @PostMapping(value = ["/internal/testExecution/markTestExecutionsOfCrashedAgentsAsFailed"])
```

Comment:
Maybe make this method not so specific, like `/testExecution/setStatusByAgentIds?status=FAILED`?

Reply:
Changed